screen/budgets/src/main/java/com/ivy/budgets/BudgetViewModel.kt

```diff
@@ -62,6 +66,8 @@ class BudgetViewModel @Inject constructor(
     private val accounts = mutableStateOf<ImmutableList<Account>>(persistentListOf())
     private val categoryBudgetsTotal = mutableDoubleStateOf(0.0)
     private val appBudgetMax = mutableDoubleStateOf(0.0)
+    private val totalRemainingBudget = mutableDoubleStateOf(0.0)
+    private val totalRemainingBudgetText = mutableStateOf<String?>("")
```

Comment:
We don't need this state - just transform the double state inside `getTotalRemainingBudgetText()`